package.json

```diff
@@ -122,8 +122,8 @@
     "@types/react": "16.7.13"
   },
   "engines": {
-    "node": ">=8.0.0 ",
-    "npm": ">=5.0.0 ",
+    "node": ">=10.0.0 ",
+    "npm": ">=6.0.0 ",
```

Comment:
Is there any requirements why a user would have to now use Node 10 instead of Node 8?

The old "node >= 8.0.0" still allows for someone to use Node 10 if they want.

I don't know if we should prevent someone from building with Node 8 just because of its current support status.

Reply:
If trying to install from fedora's package manager, f27 has v8.12, I think f28 is the same ([see](https://rpms.remirepo.net/rpmphp/zoom.php?rpm=nodejs)) and only f29 has the v10

Reply:
I assume that everyone uses package managers such as `n` or `nvm`?

Reply:
@mwringe I guess I can't understand why anyone would want to run on a maintenance version and not a LTS version. I can understand why someone wouldn't want to be on a current version or nightly build.

Reply:
>  I guess I can't understand why anyone would want to run on a maintenance version and not a LTS version

Lots of people continue to use older versions of things, either due to them not wanting to update yet or using some piece of software somewhere that still needs the older version. Node 8 isn't even end-of-life, that happens in December 2019, its just in maintenance mode. So it should still get things like critical security updates.

Lots of people also install software through the package manager on their system, and Node8 is still being used by default for a bunch of them.